src/http/AuthHttpClient.js

```diff
@@ -434,6 +434,38 @@ class AuthHttpClient {
         return this._instance(request);
     }
 
+    /**
+     * Replaces member's receipt contact.
+     *
+     * @param {Object} contact - receipt contact to set
```

Comment:
what is the object type? is it a Contact object? or is it just going to be a string?

Reply:
Contact object. Should I annotate as {Contact}? I notice other calls using {Object} for custom objects.